src/components/ReactCron/ReactCron.js

```diff
@@ -26,7 +26,7 @@ const steps = [
     title: 'Month',
     content: 'month',
   },
-];
```

Comment:
This file is auto formatted by VScode lint plugin. Just some space and string concat correction